packages/thirdweb/src/utils/signatures/sign-typed-data.test.ts

```diff
@@ -83,21 +83,6 @@ test("domain: chainId", async () => {
   );
 });
 
-test("domain: chainId hex", async () => {
```

Comment:
why the removed test?

Reply:
This was an edge case we had for wallet connect connections which I believe is resolved now, you shouldn't expect chain ID to come in as a hex